src/data.js

```diff
@@ -9,6 +9,8 @@ var jobsMarkers = {};
 // Stored with vehicle id + {_start,_end} as key
 var vehiclesMarkers = {};
 
+var overpass = {};
```

Comment:
We could avoid storing the Overpass result if it's used directly in the `ovepass.js` callback.